src/annotation/index.ts

```diff
@@ -732,7 +736,9 @@ export class Annotations {
       const undoRedo = this.redoData.pop();
       this.applyAction(undoRedo.redoAction, false);
       this.undoData.push(undoRedo);
-      this.redrawUI();
+      if (this.redrawUI !== undefined) {
+        this.redrawUI();
+      }
```

Comment:
these _can_ be written as `this.redrawUI?.()` (ie using the `?.` to check for existence)

Reply:
Done